packages/fileimport-service/src/daemon.js

```diff
@@ -81,7 +81,8 @@ async function doTask(mainDb, regionName, taskDb, task) {
       fileSize: fileSizeForMetric,
       userId: info.userId,
       streamId: info.streamId,
-      branchName: info.branchName
+      branchName: info.branchName,
+      branchId: info.branchId
```

Comment:
this will always be undefined, the info does not have a branchId field, cause the DB does not have that column